src/cryptonotecore/Currency.cpp

```diff
@@ -276,8 +281,16 @@ namespace CryptoNote
         }
 
         tx.version = CURRENT_TRANSACTION_VERSION;
-        // lock
-        tx.unlockTime = height + m_minedMoneyUnlockWindow;
+
+        uint64_t unlockTime = height + CryptoNote::parameters::CRYPTONOTE_MINED_MONEY_UNLOCK_WINDOW;
+
+        if (height >= CryptoNote::parameters::CRYPTONOTE_MINED_MONEY_UNLOCK_WINDOW_V2)
```

Comment:
Just noticed this is wrong, it should be `CryptoNote::parameters::CRYPTONOTE_MINED_MONEY_UNLOCK_WINDOW_V2_HEIGHT`

Reply:
makes sense - if height is equal or higer than CRYPTONOTE_MINED_MONEY_UNLOCK_WINDOW_V2_HEIGHT, set unllockTime as height + v2 mined money unlock window. so this, correct? : 

if (height >= CryptoNote::parameters::CRYPTONOTE_MINED_MONEY_UNLOCK_WINDOW_V2_HEIGHT)
        {
            unlockTime = height + CryptoNote::parameters::CRYPTONOTE_MINED_MONEY_UNLOCK_WINDOW_V2;
        }

Reply:
yes